src/redux/slices/cardSlices/cardsAndCardDeckSlice.js

```diff
@@ -29,17 +36,14 @@ export function loadCardDeckAndCardsByDeckId(cardDeckId) {
       const cardDeck = getCardDeckResp?.data;
       const cards = getCardsResp?.data;
       dispatch(fetchDbSuccess({cardDeck, cards}));
-
-      const storageKey = `${KEY?.RECENTLY_PLAY}/${cardDeckId}`;
-      await AsyncStorage.setItem(storageKey, JSON.stringify(cardDeck));
-      dispatch(addRecentlyKeyStore(storageKey));
-      const storeKeys = [];
-      storeKeys.push(storageKey);
-      console.log('save storeKeys: ', storeKeys);
-      await AsyncStorage.setItem(KEY.SAVE_LIB, JSON.stringify(storeKeys));
+      // dispatch(addStoreCardDecks(cardDeck));
```

Comment:
explain more reason why comment this line

Reply:
this line was used to add a pool store but in another position. However, this file is not valid because of use-redux-async-storage would be replaced this function instead.

Reply:
So remove that line instead